blazor/rich-text-editor/toolbar.md

```diff
@@ -65,21 +65,23 @@ Enable or disable the floating toolbar using [RichTextEditorToolbarSettings.Enab
 {% endhighlight %}
 {% endtabs %}
 
-![Blazor RichTextEditor with MultiRow Toolbar](./images/blazor-richtexteditor-floating.toolbar.png)
+![Blazor RichTextEditor floating toolbar](./images/blazor-richtexteditor-floating.toolbar.png)
 
-## Quick Toolbar 
+## Quick Toolbar
 
-The Rich text editor has quick toolbars that are open as context-menu on clicking the image, link, and table elements. By default, the below quick toolbar items show on click on the corresponding elements. You can customize the quick toolbar items using the [RichTextEditorQuickToolbarSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.RichTextEditorQuickToolbarSettings.html) property.
+The Rich text editor has quick toolbars that are open as context-menu on clicking the image, link, audio, video and table elements. By default, the below quick toolbar items show on click on the corresponding elements. You can customize the quick toolbar items using the [RichTextEditorQuickToolbarSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.RichTextEditorQuickToolbarSettings.html) property.
```

Comment:
Rich text editor - is it the correct casing? Rich Text Editor use this word wherever used that word

Reply:
Corrected